loader/normalize.go

```diff
@@ -75,14 +75,23 @@ func Normalize(project *types.Project, resolvePaths bool) error {
 			if s.Build.Dockerfile == "" && s.Build.DockerfileInline == "" {
 				s.Build.Dockerfile = "Dockerfile"
 			}
-			localContext := absPath(project.WorkingDir, s.Build.Context)
-			if _, err := os.Stat(localContext); err == nil {
-				if resolvePaths {
+			if resolvePaths {
+				// Build context might be a remote http/git context. Unfortunately supported "remote"
+				// syntax is highly ambiguous in moby/moby and not defined by compose-spec,
+				// so let's assume runtime will check
+				localContext := absPath(project.WorkingDir, s.Build.Context)
+				if _, err := os.Stat(localContext); err == nil {
 					s.Build.Context = localContext
 				}
-				// } else {
-				// might be a remote http/git context. Unfortunately supported "remote" syntax is highly ambiguous
-				// in moby/moby and not defined by compose-spec, so let's assume runtime will check
+				for name, path := range s.Build.AdditionalContexts {
+					if strings.Contains(path, "://") { // `docker-image://` or any builder specific context type
+						continue
+					}
+					path = absPath(project.WorkingDir, path)
+					if _, err := os.Stat(path); err == nil {
+						s.Build.AdditionalContexts[name] = path
+					}
```

Comment:
I don't think we should `os.Stat` here -- it _sorta_ makes sense for `localContext` because it's being used as a hack there to workaround the ambiguous remote syntax

Here, since we know this is a local path, we should just normalize & use it. As it stands, if you have `additional_contexts: {foo: './bar'}` and run `docker compose config, you'll get a different result based on whether `bar` exists.

Reply:
>  since we know this is a local path ..

we don't. Just like `context` this can be a git URL as well. See https://docs.docker.com/engine/reference/commandline/buildx_build/#build-context

Reply:
Aren't we handling that above?
```go
if strings.Contains(path, "://") { // `docker-image://` or any builder specific context type
```
Or is it possible to specify a Git URL without a protocol?? (e.g. `github.com/docker/compose.git` without `https://` or `git://` etc)

Reply:
git URL might indeed not have a `scheme://` prefix, 
especially, we support `github.com/xx/ff`  for legacy reasons. See https://github.com/moby/buildkit/blob/master/util/gitutil/git_ref.go#L54

Reply:
Ahhh ok I did not know that and I'm terrified now